weave-js/src/components/PagePanelComponents/Home/Browse3.tsx

```diff
@@ -736,7 +735,7 @@ const CallsPageBinding = () => {
     try {
       return JSON.parse(query.cols);
     } catch (e) {
-      return DEFAULT_COLUMN_VISIBILITY_CALLS;
+      return {};
```

Comment:
why this change? Seems unrelated?

Reply:
This is actually related, we now have more fields we want to keep hidden, and they are dynamically generated so we can't rely on a constant. This handling is now moved into `callsTable`

Reply:
i see - this was changed in the use effect